lib/config.json

```diff
@@ -4,6 +4,7 @@
         "port": "execBase:./utils#resolvePort",
         // resolve host based on env var HOSTNAME, falls back to default '0.0.0.0'
         "host": "execBase:./utils#resolveHost",
+        "debug": false,
```

Comment:
This shouldn't be set unless all environments use transports. Only development is currently set up for that.

Reply:
yeah!! is better to use it directly when all the envs have their configuration